server/src/main/scala/za/co/absa/atum/server/api/database/runs/functions/GetPartitioningCheckpoints.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.database.runs.functions
+
+import doobie.Fragment
+import doobie.implicits.toSqlInterpolator
+import doobie.util.Read
+import play.api.libs.json.Json
+import za.co.absa.atum.model.dto.CheckpointQueryDTO
+import za.co.absa.atum.server.api.database.PostgresDatabaseProvider
+import za.co.absa.atum.server.model.{CheckpointMeasurements, PartitioningForDB}
+import za.co.absa.fadb.DBSchema
+import za.co.absa.fadb.doobie.DoobieEngine
+import za.co.absa.fadb.doobie.DoobieFunction.DoobieMultipleResultFunction
+import zio._
+import zio.interop.catz._
+import za.co.absa.atum.server.api.database.runs.Runs
+import za.co.absa.atum.server.api.database.DoobieImplicits.Sequence.get
+import doobie.postgres.implicits._
+import doobie.postgres.circe.jsonb.implicits._
+import io.circe.syntax.EncoderOps
+import io.circe.generic.auto._
+
+class GetPartitioningCheckpoints (implicit schema: DBSchema, dbEngine: DoobieEngine[Task])
+  extends DoobieMultipleResultFunction[CheckpointQueryDTO, CheckpointMeasurements, Task] {
+
+  override val fieldsToSelect: Seq[String] = Seq(
+    "id_checkpoint",
+    "checkpoint_name",
+    "author",
+    "measured_by_atum_agent",
+    "measure_name",
+    "measure_columns",
```

Comment:
```suggestion
    "measured_columns",
```

Reply:
and to all used/relevant places - to be consistent with the rest of the repo

Reply:
Okay thanks.